@types/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: any
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4694377)